cmd/agent/main.go

```diff
@@ -186,6 +193,35 @@ func cleanup(ctx context.Context) error {
 	return nil
 }
 
+type RkeKubeletIPAddress struct {
```

Comment:
Might be a good idea to make this an interface for testing purposes.

Reply:
I added an interface, though I can't find much more to add unit tests for since this updated logic is focused on communication with the node-server and really just does file IO. The current tests cover all cases where a cert may be invalid due to changes in IP addresses, hostnames, expiration dates